src/main/java/slimeknights/tconstruct/library/materials/stats/IMaterialStats.java

```diff
@@ -8,9 +8,6 @@
  */
 public interface IMaterialStats {
 
-  /**
-   * Returns a unique ResourceLocation to identify the type of stats the material has.
-   */
```

Comment:
Any reason for this removal?

Reply:
..no idea why/how that got removed tbh :D